versions/3.1.1.md

```diff
@@ -314,12 +314,24 @@ This is the root object of the [OpenAPI document](#openapi-description).
+| <a name="oas-security"></a>security                    |  [[Security Requirement Object](#security-requirement-object)]  | See [the `security` field](#oas-security-field).
 | <a name="oas-tags"></a>tags                            |                  [[Tag Object](#tag-object)]                   | A list of tags used by the document with additional metadata. The order of the tags can be used to reflect on their order by the parsing tools. Not all tags that are used by the [Operation Object](#operation-object) must be declared. The tags that are not declared MAY be organized randomly or based on the tools' logic. Each tag name in the list MUST be unique.                                                                                                                                                                               |
 | <a name="oas-external-docs"></a>externalDocs            | [External Documentation Object](#external-documentation-object) | Additional external documentation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 
 This object MAY be extended with [Specification Extensions](#specification-extensions).
 
+###### <a name="oas-security-field"></a>The `security` field
+
+The `security` field describes how requests are authorized:
+
+- If omitted, then nothing can be inferred about the authorization requirements; the behaviour is implementation-defined.
+- If present but empty (`security: []`), then the behaviour is undefined.
+- Otherwise, it is an array of Security Requirement Objects, only one of which needs to be satisfied for the request to be authorized.
+
+Because the empty [Security Requirement Object](#security-requirement-object) `{}` will always be satisfied, any `security` list that includes `{}` will allow all requests. In particular, `security: [{}]` means that no security schemes are in use (also known as "no security").
```

Comment:
```suggestion
Because the empty Security Requirement Object `{}` will always be satisfied, any `security` list that includes `{}` will allow all requests. In particular, `security: [{}]` means that no security schemes are in use (also known as "no security").
```